ui/admin/app/components/agent/FirstModelProviderBanner.tsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from "@remix-run/react";
+import { $path } from "remix-routes";
+
+import { TypographyP } from "~/components/Typography";
+import { OttoLogo } from "~/components/branding/OttoLogo";
+import { Button } from "~/components/ui/button";
+
+export function FirstModelProviderBanner() {
+    return (
+        <div className="w-full">
+            <div className="flex justify-center w-full">
+                <div className="flex flex-row p-4 min-h-36 justify-end items-center w-[calc(100%-4rem)] rounded-sm mx-8 mt-4 bg-secondary relative overflow-hidden gap-4 max-w-screen-md">
+                    <OttoLogo
+                        hideText
+                        classNames={{
+                            root: "absolute opacity-45 top-[-5rem] left-[-7.5rem]",
+                            image: "h-80 w-80",
+                        }}
+                    />
+                    <div className="flex flex-col pl-48">
+                        <TypographyP className="font-semibold text-2xl mb-0.5">
+                            Ready to create your first Agent?
+                        </TypographyP>
```

Comment:
can you use a `TypographyH_` here the primary goal of the `Typography` components is to reuse styles for `font-weight`, `font-size`, etc. It's also helpful for screen readers to use proper headings where possible